src/biokbase/narrative/tests/test_updater.py

```diff
@@ -14,31 +14,20 @@ def __init__(self, keyname, source):
 
 class UpdaterTestCase(unittest.TestCase):
     @classmethod
-    def setUpClass(self):
+    def setUpClass(cls):
```

Comment:
these are minor flake8 / pylance errors that I fixed whilst in the neighbourhood